src/Tribe/Service_Providers/Widgets.php

```diff
@@ -0,0 +1,87 @@
+<?php
+namespace Tribe\Service_Providers;
+
+use Tribe\Widget\Manager;
+
+/**
+ * Class Widget
+ *
+ * @since   tBD
+ *
+ * @package Tribe\Service_Providers
+ */
+class Widgets extends \tad_DI52_ServiceProvider {
+
+	/**
+	 * Binds and sets up implementations.
+	 *
+	 * @since tBD
+	 */
+	public function register() {
+		if ( ! static::is_active() ) {
+			return;
+		}
+
+		$this->container->singleton( Manager::class, Manager::class );
+		$this->container->singleton(
+			'widget.manager',
+			function() {
+				return $this->container->make( Manager::class );
+			}
+		);
+
+		$this->register_hooks();
+		$this->register_assets();
+
+		$this->container->singleton( static::class, $this );
+
+	}
+
+	/**
+	 * Static method wrapper around a filter to allow full deactivation of this provider
+	 *
+	 * @since tBD
+	 *
+	 * @return boolean If this service provider is active.
+	 */
+	public static function is_active() {
+		/**
+		 * Allows filtering to deactivate all widgets loading.
+		 *
+		 * @since tBD
+		 *
+		 * @param boolean $is_active If widgets should be loaded or not.
+		 */
+		return apply_filters( 'tribe_widgets_is_active', true );
+	}
+
+	/**
+	 * Register all the assets associated with this service provider.
+	 *
+	 * @since tBD
+	 */
+	protected function register_assets() {
+
```

Comment:
Any assets?

Reply:
I am going to remove these, no assets yet.